CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 
 # Unreleased (plans for 1.3.0)
 
+* Remove the remaining Rails UJS integration in favor of Turbo-powered helpers.
+* Migrate from Turbolinks to Hotwire Turbo
```

Comment:
This will be a breaking change for users who are on Turbolinks so we should release as 2.0.

Reply:
it's not that the support is dropped, it's not needed anymore.

Reply:
that's true, but we drop the configuration to allow people to use jquery-ujs instead of rails-ujs (some existing users may be still using jquery-ujs rather than rails-ujs - potentially)

Reply:
added note of it being removed

Reply:
Sorry I only just got what you meant - see longer comment elsewhere